core/src/main/scala/org/broadinstitute/dsde/rawls/user/UserService.scala

```diff
@@ -329,15 +335,21 @@ class UserService(protected val userInfo: UserInfo, val dataSource: SlickDataSou
               }
             }
 
-            createProjectOperation <- gcsDAO.createProject(projectName, billingAccount).recoverWith {
+            //NOTE: we're syncing this to Sam ahead of the resource actually existing. is this fine? (ps these are sam calls)
```

Comment:
the resource exists in sam but the project has not been created in google yet... should be fine